cmd/ncproxy/ncproxy.go

```diff
@@ -463,6 +463,22 @@ func (s *grpcService) AddEndpoint(ctx context.Context, req *ncproxygrpc.AddEndpo
 			}
 			return nil, errors.Wrapf(err, "failed to get endpoint with name `%s`", req.Name)
 		}
+		if req.AttachToHost {
+			if req.NamespaceID != "" {
```

Comment:
Could have used NamespaceID == "" to imply attach to host since it was previously an error